hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/records/CreateFileRecordBuilderImpl.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.file.impl.records;
+
+import com.hedera.node.app.spi.records.UniversalRecordBuilder;
+
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+public class CreateFileRecordBuilderImpl extends UniversalRecordBuilder<CreateFileRecordBuilder>
+        implements CreateFileRecordBuilder {
+    private long createdFileNum = 0;
+
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    protected CreateFileRecordBuilderImpl self() {
```

Comment:
What is the value of this method over `this`?

Reply:
UniversalBuilderRecord requires it